.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -0,0 +1,140 @@
+## Description
+_Please provide a summary of the release and the issue it fixes. 
+Please add necessary details, context, including version numbers._
+
+## Component Checklists
+_Each maintainer, please fill out the necessary in formation and use check box to check the progress._
+
+### incore-auth
+- Maintainer: Yong Wook Kim
+- Version number:
+- [ ] <strong>This will not be released in this release</strong>
```

Comment:
You can replace <strong> </strong> with **
```suggestion
- [ ] **This will not be released in this release**
```

Reply:
replaced